cli/format.go

```diff
@@ -35,6 +36,9 @@ func (f *Format) Run() (err error) {
 	// set log level and other options
 	f.configureLogging()
 
+	// initialise stats collection
+	stats.Init()
```

Comment:
Moved this earlier as it wasn't being reset correctly in tests if an error was thrown.